README.md

```diff
@@ -5,14 +5,15 @@
 
 [kotlin-inject](https://github.com/evant/kotlin-inject) is a compile-time dependency injection
 framework for Kotlin Multiplatform similar to Dagger 2 for Java.
-[Anvil](https://github.com/square/anvil) extends Dagger 2 to simplify dependency injection. This
-project provides a similar feature set for the `kotlin-inject` framework.
+[Anvil](https://github.com/square/anvil) extends Dagger 2 to simplify dependency injection.
+
+This project provides a similar feature set for the `kotlin-inject` framework. The extensions provided
+by `kotlin-inject-anvil` allow you to contribute and automatically merge component interfaces without explicit
+references in code.
 
-The extensions provided by `kotlin-inject-anvil` allow you to contribute and automatically merge
-component interfaces without explicit references in code.
 ```kotlin
 @ContributesTo(AppScope::class)
-interface AppIdComponent {
+interface NameComponent {
```

Comment:
the thinking here is not to confuse the first time reader that `AppIdComponent` has anything to do with `AppComponent`

Reply:
It makes it inconsistent with the method below. I'd like to keep the original name.